pym/bob/cmds/build.py

```diff
@@ -1082,6 +1132,13 @@ def __getCheckoutStepBuildId(self, step, depth):
 
         return ret, predicted
 
+    def _checkSharedPackage(self, step, packageBuildId):
+         sharedPackagePath = os.path.join(self.__sharedFolder, packageBuildId)
+         if os.path.isdir(sharedPackagePath):
+             if os.path.exists(os.path.join(sharedPackagePath, '.installed')):
```

Comment:
The ".installed" canary is not needed if the folder is atomically "installed" (renamed) to the right place.